qemu/tests/cfg/boot_from_device.cfg

```diff
@@ -73,19 +73,19 @@
                     bootindex_image1 = 1
                 - with_specify_device:
                     boot_device = "DVD/CD"
-        - boot_from_iscsi_device:
-            start_vm = no
-            dev_name = iscsi-dev
-            portal_ip = 10.66.90.100
-            initiator = "iqn.2010-07.com.redhat:kvmautotest"
-            target = "iqn.2001-05.com.equallogic:0-8a0906-db31f7d03-470263b05654c204-kvm-puyiqiao"
-            images = "stg"
-            image_name_stg = "/dev/sdb"
-            image_format_stg = ""
-            drive_format_stg = scsi-block
-            variants:
-                - with_remote_stg:
-                    bootindex_stg = 0
-                - with_specify_device:
-                    boot_device = "virtio-scsi Drive"
+        #- boot_from_iscsi_device:
+        #    start_vm = no
+        #    dev_name = iscsi-dev
+        #    portal_ip = 4.2.2.1
+        #    initiator = "iqn.2010-07.com.example:my_initiator"
+        #    target = "iqn.2001-05.com.mytarget:0-0a0000-000000000-0000000000000000-my-target
```

Comment:
Commenting them out would result in the test being performed locally, without iscsi dev. IMO the commented version is the best solution, but I'd also be fine keeping it here but with dummy setting like portal_ip=127.0.0.1, which should fail trying to contact the initiator. What do you think?

Reply:
I'd prefer keeping the dummy here, so that we can pass the values through avocado's parameter to run this test without modification of the source code.

Reply:
Sure, I'll use some obviously broken values (`like "initiator.com.examle:provide_your_initiator"`) in v2